### PR TITLE
make check: add ineffassign

### PR DESCRIPTION
**Pull request by tamird**

### CODE REVIEW

**Comment by BramGruneir** [comment]
LGTM

File: sql/parser/normalize.go [hunk 1]
```diff
@@ -399,7 +399,7 @@ func (v *isConstVisitor) Visit(expr Expr, pre bool) (Visitor, Expr) {
 // (i.e. it does not contain a VariableExpr, QualifiedName, etc).
 func isConst(expr Expr) bool {
 	v := isConstVisitor{isConst: true}
-	expr = WalkExpr(&v, expr)
+	_ = WalkExpr(&v, expr)
```
Comment (bdarnell) [inline sql/parser/normalize.go]: Does `ineffassign` require these dummy assignments?

**Comment by bdarnell** [comment]
LGTM

File: sql/parser/normalize.go (hunk 1 above)
Comment (tamird) [inline sql/parser/normalize.go]: no, that was a judgement call. I preferred to keep the indication that the function returns a value.

**Pull request merged**